specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobAgents.json

```diff
@@ -45,7 +45,7 @@
             }
           },
           "default": {
```

Comment:
>default [](http://example.com/codeflow?start=11&length=7)

you should also declare the schema of the default [error] response. It should be declared as a `$ref` to ErrorResponse here

[https://github.com/Azure/azure-rest-api-specs/blob/32c178f2467f792a322f56174be244135d2c907f/specification/common-types/resource-management/v5/types.json#L303](https://github.com/Azure/azure-rest-api-specs/blob/32c178f2467f792a322f56174be244135d2c907f/specification/common-types/resource-management/v5/types.json#L303)

Reply:
These seems to have been autogenerated by engine by list of ErrorDescriptions

Reply:
I don't know what engine you're using, but could you figure out a way to fix, or further configur eit?